src/link/patch.cpp

```diff
@@ -267,7 +267,7 @@ static int32_t computeRPNExpr(Patch const &patch, std::vector<Symbol> const &fil
 
 		case RPN_BANK_SELF:
 			if (!patch.pcSection) {
-				error(patch.src, patch.lineNo, "PC has no bank outside a section");
+				error(patch.src, patch.lineNo, "PC has no bank outside of a section");
```

Comment:
Perhaps we should reword that to “PC doesn't have a bank outside of a section”?

Reply:
Eh, let's do a separate PR for considering more drastic message rephrasings. I just wanted to make this one consistent with others that use "of".